components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -36,6 +36,7 @@ export type InfoListItemProps = Omit<Omit<ListItemProps, 'title'>, 'divider'> &
     statusColor?: string;
     subtitle?: string | Array<string | JSX.Element>;
     subtitleSeparator?: string;
+    info?: string | Array<string | JSX.Element>;
```

Comment:
Should these `<string | JSX.Element>` be just ReactNodes? @joebochill

Reply:
Tom and I discussed this a little earlier. I think there's a bigger refactor coming for the subtitle where we will just accept a `ReactNode` and get rid of all of the internal logic for adding in the separator. Since that would be a breaking change, we decided to leave it as is for the moment and let the `info` prop match the implementation/usage of subtitle.